### DISCUSSION TITLE
用新版noahmp跑作物的时候，但是作物产量始终是0。

### DISCUSSION THREAD

**Discussion by 12wlj** [NCAR/hrldas#188] [General]
用新版noahmp跑作物的时候，模型成功运行了，但是作物产量始终是0，别的变量有数值，我试了几个作物类型，都没有产量输出，该打开的作物选项打开了，有没有大神帮帮忙。
是我做驱动时候的问题吗

**Comment by cenlinhe** [comment]
Hi, this may be because your simulation period is not in the growing season or the planting date input or parameter is not set correctly. please double check your planting date. also make sure that crop_option = 1 in the namelist and crs_option=1.

**Reply by 12wlj** [reply]
crop_option = 1 这个选项我打开了， crs_option=1这个我没找到，我试一下，谢谢啦

**Reply by 12wlj** [reply]
> crop_option = 1 这个选项我打开了， crs_option=1这个我没找到，我试一下，谢谢啦

**Reply by cenlinhe** [reply]
CANOPY_STOMATAL_RESISTANCE_OPTION = 1

**Comment by cenlinhe** [comment]
@CharlesZheZhang

**Reply by 12wlj** [reply]
您好，我再问一下，跑作物模型的时候，DYNAMIC_VEG_OPTION                = 4，这里设置为几呢，2还是4，还是5，请您帮我解答一下疑惑。

**Reply by CharlesZheZhang** [reply]
Hi, you can keep using DYNAMIC_VEG_OPTION = 4 (these options are for non-cropland grids) when you use crop_option=1.

**Reply by 12wlj** [reply]
谢谢您的回复，我需要用农田网格的话，是要开启DYNAMIC_VEG_OPTION别的选项吗，我需要跑小麦作物，这里是不是要用别的选项？期盼您的解答，谢谢。

**Comment by CharlesZheZhang** [comment]
Hi, are you running single-point simulation, like the case in [Bondville? ](https://github.com/NCAR/hrldas/blob/master/tutorial/Note1_Single_Point.ipynb)

To run the crop model, make sure to:
1. turn on crop option crop_option=1
2. the land use of this grid point is cropland, LU_INDEX=12 in MODIS land use type
3. change DEFAULT_CROP to the crop species (1 or 2 for corn or soybean) in [NoahmpTable.TBL](https://github.com/NCAR/noahmp/blob/master/parameters/NoahmpTable.TBL)

**Reply by 12wlj** [reply]
> 嗨，您是否正在运行单点仿真，就像[邦德维尔的情况一样？ ](https://github.com/NCAR/hrldas/blob/master/tutorial/Note1_Single_Point.ipynb)
> 
> 要运行裁剪模型，请确保：
> 
> 1. 打开裁剪选项 crop_option=1
> 2. 该网格点的土地利用为耕地，在MODIS土地利用类型中LU_INDEX=12
> 3. 更改 [NoahmpTable.TBL](https://github.com/NCAR/noahmp/blob/master/parameters/NoahmpTable.TBL) 中的作物种类（玉米或大豆为 1 或 2）的DEFAULT_CROP

谢谢您的帮助，我已经解决问题了，再次感谢！
